### DISCUSSION TITLE
Example of Homogeneous Link Regression

### DISCUSSION THREAD

**Discussion by JohnComo** [pyg-team/pytorch_geometric#8307] [Q&A]
Hi everyone, I see a lot of examples for link prediction notebooks, I am reaching out to see if there any examples of link regression for homogeneous graphs. Thanks in advance !

**Comment by rusty1s** [comment]
Currently no, but the MovieLens example is a heterogeneous link regression use-case, so it follows a similar style than doing that on homogeneous graphs. Let me know if you wanna discuss further on how to do it :)
